### DISCUSSION TITLE
MAST implementation of Seurat

### DISCUSSION THREAD

**Discussion by cv55** [satijalab/seurat#6162] [General]
Hello, thanks for the great tool!

I want to do DE analysis on my data, which consists of controls and patients. It seems that even after normalising and transforming the data the controls have overall higher gene expression (I am estimating the average of each cell from the data slot). So i though this is something I should take into account when performing the DE and decided to go for the MAST implementation, since it allows correction for latent variables. I was wondering does Seurat regress out the cellular detection rate by default when running MAST, or do I have to estimate it myself and pass it in as a latent variable?

Thanks,
Christos

**Comment by RaghadShu** [comment]
Hi @cv55, have you found the answer to this?
